src/index/Index.cpp

```diff
@@ -1312,7 +1312,6 @@ void Index::scanOSP(const string& object, IdTable* result) const {
 void Index::scanPSO(Id predicate, IdTable* result) const {
```

Comment:
Nice finding, but you probably will run into trouble after my reworking of the index class.
Maybe cancel this change, then rebase, then add it again:)